src/System.Management.Automation/engine/CommandCompletion/CommandCompletion.cs

```diff
@@ -180,17 +177,11 @@ public static CommandCompletion CompleteInput(string input, int cursorIndex, Has
                 if (!powershell.IsChild)
                 {
                     CheckScriptCallOnRemoteRunspace(remoteRunspace);
+
                     if (remoteRunspace.GetCapabilities().Equals(Runspaces.RunspaceCapability.Default))
                     {
-                        // Remoting to a Win7 machine. Use the legacy tab completion function from V1/V2
-                        int replacementIndex;
-                        int replacementLength;
-
-                        powershell.Commands.Clear();
-                        var results = InvokeLegacyTabExpansion(powershell, input, cursorIndex, true, out replacementIndex, out replacementLength);
-                        return new CommandCompletion(
-                            new Collection<CompletionResult>(results ?? EmptyCompletionResult),
-                            -1, replacementIndex, replacementLength);
+                        // Remoting to a Win7 machine.
```

Comment:
Technically a Win7 machine can have 5.1 installed via WMF5.1, so this comment should really be remoting to a PS2.0 machine